src/components/Hero/Hero.jsx

```diff
@@ -44,21 +45,59 @@ const handleDelete = (id) => {
 function Hero() {
   const [type, setType] = useState("expenses");
 
-//   const handleTabChange = (selectedType) => {
-//     setType(selectedType);
-//   };
+  const handleTabChange = (selectedType) => {
+    setType(selectedType);
+  };
+
   return (
-    <div className={css['hero-wrapper']}>
-      <FormTransaction />
-      <div className={css['hero-table']}>
-      <TransactionTable
-        transactions={transactions}
-        type={type}
-        handleDelete={handleDelete}
-      />
-      <SummaryTable data={data} /></div>
+      <div className={css["hero-container"]}>
+      <nav className={css["tab-wrapper"]}>
+        <NavLink
+          to="/expenses"
+          className={type === "expenses" ? css["tab-active"] : css["tab"]}
+          onClick={() => handleTabChange("expenses")}
+        >
+          Expenses
+        </NavLink>
+
+        <NavLink
+          to="/income"
+          className={type === "income" ? css["tab-active"] : css["tab"]}
+          onClick={() => handleTabChange("income")}
+        >
+          Income
+        </NavLink>
+      </nav>
+
```

Comment:
uwaga bo to będzie pogmatwane
routy usuwasz z tego pliku i wklejasz takie coś jak outlet, nie jestem pewien czy w hero wraper czy jego też będziesz musiała przenieść ale to zaraz wyjaśnie niżej
![image](https://github.com/ToporekDaniel/kapusta-app/assets/138520262/5da2c3e4-5d1f-45f4-a68b-4be6ad51b094)
tu jest przykład ze strarego zadania gdzie linki odpowiadają twoim navlinkom a w outlecie wyświetla się reszta